server-http-handler/execute-requests.go

```diff
@@ -49,8 +49,7 @@ func main() {
 	executionID := os.Args[4]
 	jarPath := os.Args[5]
 	handlerType := os.Args[6]
-	trace := os.Args[7]
-	optPath := os.Args[8]
+	optPath := os.Args[7]
```

Comment:
Ao menos checar se os argumentos estão corretos? Um mínimo de checagens de sanidade faz bem na hora que estamos tentando usar essas coisas.

Reply:
Done! Thanks for the tips!